### PR TITLE
Universal dmx port

### PR DESCRIPTION
**Pull request by ByronAmbright**
This is the work in progress for getting rid of the hardcoded values, and make dmx universal between the pages.

### CODE REVIEW

**Comment by bookdude13** [comment]
This replaces #72

**Comment by bookdude13** [comment]
@ByronAmbright I can't make you a reviewer, since you opened this PR. @theeldestelder this is ready to be looked at

**Comment by ByronAmbright** [comment]
@bookdude13 @theeldestelder  I changed a few things up and added the tim combination in this pull request. all you should have to do is open the directory of sequences on the combine page, then it should combine the entire show. This is a little specific to our show this year, but should be made individual to each show we make in the future. @bookdude13  I also changed how the collapse button works, let me know what you think.

**Comment by bookdude13** [comment]
I like the change, and tweaked it further to add some margin on the left for the dmx selector. I also pushed some lint fixes - there still needs some work done on that front

**Comment by bookdude13** [comment]
Disabling linting in timCombination.js for now. The fields in the .tim files don't follow our conventions, and most of that file was failing the linter when it could be argued it shouldn't.

File: app/js/timCombination.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
+
+const path = parent.require('path');
+const xmlReader = parent.require('xml2js');
+const fs = parent.require('fs');
+
+/*
```
[the remark this answers is not in the document]
Comment (theeldestelder) [reply app/js/timCombination.js]: What do you mean here?  These libraries are used below.  I'll get rid of the commented out portion immediately below this.

**Comment by theeldestelder** [comment]
I agree with the feedback to the user about where things are written to and disabling the buttons.  I was working on a version of that in a different branch for better feedback but something should be here.  I'll work on this now.

**Comment by theeldestelder** [comment]
I added a message indicating which folder contains the combined sequences.  Everything works on my end and tested with working show

**Pull request merged**